### ISSUE TITLE
Include the uri in the Status List Token?

### ISSUE THREAD

**Issue by paulbastian** [vcstuff/draft-looker-jwt-cwt-status-list#28]
If a verifier fetched the Status List Token and checked the status of the Referenced Token, he might want to store both of those, e.g. for archiving or regulative requirements.
IF somebody were to review the data later, there is currently no guarantee that these matched together in the past, other than ìss` is the same for both.
I just thought about, if it makes sense to include the uri or some part of identifier in the Status List Token itself?

**Comment by c2bo** [comment]
That is also missing for the offline case (bring your status list with you) to make sure we have the correct status list. 

Imho, we can just use the subject claim and bind it to the URI that is also used in the Referenced Token? 
A status list JWT would then look like this:
```
{
  "sub": "https://example.com/statuslists/1", 
  "iss": "https://example.com",
  "iat": 1683560915,
  "exp": 1686232115,
  "status_list": {
    "bits": 1,
    "lst": "H4sIAAAAAAAAA-3BMQEAAADCoPVPbQwfoAA......IC3AYbSVKsAQAAA"
  }
}
```
What do you think?

**Comment by paulbastian** [comment]
yes, this is what I thought. `sub` is probably the best choice

**Comment by tplooker** [comment]
Resolved in #30
